platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/crypto/EnhancedKeyStoreLoader.java

```diff
@@ -1476,7 +1476,7 @@ private void cleanupByMovingPfxFilesToSubDirectory() throws KeyStoreException, K
         if (sPublicPfx.exists()
                 && sPublicPfx.isFile()
                 && !sPublicPfx.renameTo(
-                        pfxArchiveDirectory.resolve(sPublicPfx.getName()).toFile())) {
+                        pfxDateDirectory.resolve(sPublicPfx.getName()).toFile())) {
```

Comment:
I notice that the variable name is incorrect, `pfxDateDirectory` instead of `pfxDataDirectory`

Up to you if you want to fix it in this PR

Reply:
Date directory is correct.  The directory is literally the date.